schema/snapcraft.json

```
@@ -917,6 +917,7 @@
                                     "gnome-3-38",
                                     "kde-neon",
                                     "kde-neon-6",
+                                    "kde-neon-qt6"
```

```suggestion
                                    "kde-neon-qt6",
```
This should fix a lot of your legacy test failures